plasmapy/utils/decorators/tests/test_validators.py

```diff
@@ -555,3 +560,76 @@ def test_decorator_func_def(self, mock_vq_class):
                 # reset
                 mock_vq_class.reset_mock()
                 mock_foo.reset_mock()
+
+
+class TestValidateClassAttributes:
```

Comment:
I don't feel qualified to review this decorator - I suggest moving it to a separate PR and then maybe @namurphy can take a look at it?

Reply:
I can definitely sign off on it 😉 The test looks reasonable at least, still digging into the implementation.